src/named_threads.cc

```diff
@@ -49,7 +49,13 @@ void set_current_thread_name(const thread_type t_type);
 void name_and_pin_and_order_this_thread(const thread_type t_type) {
 	set_current_thread_name(t_type);
 	thread_pinning::pin_this_thread(t_type);
-	CELERITY_DETAIL_TRACY_SET_THREAD_NAME_AND_ORDER(t_type);
+
+#if CELERITY_TRACY_SUPPORT
+	const char* name = tracy_detail::leak_name(named_threads::thread_type_to_string(t_type));
+	const int32_t order = tracy_detail::lane_order::thread + static_cast<int32_t>(t_type);
+	assert(order <= static_cast<int32_t>(tracy_detail::lane_order::thread_max));
+	::tracy::SetThreadNameWithHint(name, order);
+#endif
```

Comment:
I would prefer if this remained contained in the tracy module.

Reply:
I found a way to move this code back to tracy.h by stringifying / casting `thread_type` in `named_threads`, and applying them in `tracy.h`.